.webpack/webpack.config.base.js

```diff
@@ -35,7 +37,11 @@ export default {
         test: /\.js$/,
         include: [resolveRoot('./src'), resolveRoot('./node_modules/dagre-d3-renderer/lib')],
         use: {
-          loader: 'babel-loader',
+          loader: 'esbuild-loader',
+          options: {
+            implementation: esbuild,
+            target: 'es2015',
```

Comment:
Should we target ES6 instead?

Reply:
What is the difference? From what I could find, it's the same

Reply:
My bad. Subconsciously related ES201`5` to ES`5`.  🤦‍♂️